dev/com.ibm.ws.cdi.jndi_fat/test-applications/jndiLookup.war/src/com/ibm/ws/cdi12/test/jndi/BeanManagerP.java

```diff
@@ -8,28 +8,30 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package com.ibm.ws.cdi12.fat.injectparameters;
+package com.ibm.ws.cdi12.test.jndi;
 
 import java.io.IOException;
+import java.io.PrintWriter;
 
-import javax.ejb.EJB;
-import javax.servlet.ServletException;
+import javax.inject.Inject;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.ibm.ws.cdi12.test.jndi.observer.ObserverBean;
 
-@SuppressWarnings("serial")
-@WebServlet("/TestEjb")
-public class TestEjbServlet extends HttpServlet {
+import javax.annotation.Resource;
+import javax.enterprise.inject.spi.BeanManager;
+import javax.enterprise.inject.Produces;
+import javax.enterprise.context.ApplicationScoped;
 
-    @EJB
-    TestEjb ejb;
+import javax.inject.Named;
 
-    @Override
-    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
-                    throws ServletException, IOException {
-        resp.getOutputStream().println(TestUtils.join(ejb.getResult()));
-    }
+@ApplicationScoped
+public class BeanManagerP  {
+
+    @Produces
+    @Resource(lookup = "java:comp/BeanManager")
+    private String manager;
 
 }
```

Comment:
This appears to be a new class (even though github is showing it as having moved from another project). What is it for? I don't see a corresponding change in this project where a test would make use of it.

Reply:
Agree, this looks misplaced? @benjamin-confino

Reply:
I agree. It can be deleted.